app/views/episodes/_episode.html.erb

```diff
@@ -22,7 +22,11 @@
           <% end %>
         </div>
 
-        <%= link_to "Go to Episode", edit_episode_path(episode.guid), class: "stretched-link", data: {turbo_frame: "_top"} %>
+        <% if Rails.env.development? && clickhouse_connected? && episode.published? %>
+          <%= link_to "Go to Episode", episode_metrics_path(episode.guid), class: "stretched-link", data: {turbo_frame: "_top"} %>
```

Comment:
link to `overview` tab for published episodes, and regular "settings" tab for drafts.